hydra-main/src/main/java/com/addthis/hydra/job/store/DataStoreUtil.java

```diff
@@ -109,6 +110,11 @@ public static SpawnDataStore makeSpawnDataStore(DataStoreType type) throws Excep
         Properties properties = new Properties();
         if (sqlUser != null) {
             properties.put("user", sqlUser);
+            if(sqlPasspath.length() != 0) {
+                VaultWrapper vaultClient = (VaultWrapper) Class.forName("com.addthis.hydra.vault.VaultWrapperImpl").newInstance();//new VaultWrapperImpl();
```

Comment:
Hard coding a classname that only exists in closed source is not the right way to do this.